applications/GeoMechanicsApplication/tests/test_thermal_element/test_micro_climate_2/ProjectParameters.json

```diff
@@ -0,0 +1,209 @@
+        }
+      }
+    },{
+      "python_module"                    : "gid_output_process",
+      "kratos_module"                    : "KratosMultiphysics",
+      "process_name"                     : "GiDOutputProcess",
+      "Parameters" : {
+        "model_part_name"                : "PorousDomain.PointThermo",
+        "output_name"                    : "test_micro_climate_2_his",
+        "postprocess_parameters" : {
+          "result_file_configuration": {
+            "gidpost_flags" : {
+              "WriteDeformedMeshFlag"    : "WriteUndeformed",
+              "WriteConditionsFlag"      : "WriteElementsOnly",
+              "GiDPostMode"              : "GiD_PostAscii",
+              "MultiFileFlag"            : "SingleFile"
+            },
+            "file_label"                 : "step",
+            "output_control_type"        : "step",
+            "output_interval"            : 1,
+            "body_output"                : true,
+            "node_output"                : false,
+            "skin_output"                : false,
+            "plane_output"               : [],
+            "nodal_results"              : ["TEMPERATURE"],
+            "gauss_point_results"        : []
+          },
+          "point_data_configuration"     : []
+        }
+      }
+    }]
+  },
+  "processes": {
+    "constraints_process_list" : [{
+      "python_module"                    : "apply_scalar_constraint_table_process",
+      "kratos_module"                    : "KratosMultiphysics.GeoMechanicsApplication",
+      "process_name"                     : "ApplyScalarConstraintTableProcess",
+      "Parameters" : {
+        "model_part_name"                : "PorousDomain.StartThermo",
+        "variable_name"                  : "TEMPERATURE",
+        "is_fixed"                       : false,
+        "value"                          : 10,
+        "table"                          : 0
+      }
+    },{
+      "python_module"                    : "apply_scalar_constraint_table_process",
+      "kratos_module"                    : "KratosMultiphysics.GeoMechanicsApplication",
+      "process_name"                     : "ApplyScalarConstraintTableProcess",
+      "Parameters" : {
+        "model_part_name"                : "PorousDomain.BotThermo",
+        "variable_name"                  : "TEMPERATURE",
+        "is_fixed"                       : true,
+        "value"                          : 10,
+        "table"                          : 6
+      }
+    }],
+    "loads_process_list" : [{
+      "python_module"                    : "apply_scalar_constraint_table_process",
+      "kratos_module"                    : "KratosMultiphysics.GeoMechanicsApplication",
+      "process_name"                     : "ApplyScalarConstraintTableProcess",
+      "Parameters" : {
+        "model_part_name"                : "PorousDomain.TopThermo",
+        "variable_name"                  : "AIR_TEMPERATURE",
```

Comment:
can we output and display these new variables? that would be a natural addition for model checking.